draft-ietf-mmusic-sdp-uks.md

```diff
@@ -336,14 +342,56 @@ Section 3 of {{!TLS13}}:
    } ExternalIdentityHash;
 ~~~
 
-A WebRTC identity assertion is provided as a JSON {{?JSON=RFC8259}} object that
-is encoded into a JSON text.  The resulting string is then encoded using UTF-8
-{{!UTF8=RFC3629}}.  The content of the `external_id_hash` extension are produced
-by hashing the resulting octets with SHA-256 {{!SHA=RFC6234}}.  This produces
-the 32 octets of the `binding_hash` parameter, which is the sole contents of the
+Where an identity assertion has been asserted by a peer, this extension includes
+a SHA-256 hash of the assertion.  An empty value is used to indicates support for
+the extension.
```

Comment:
maybe "support for the extension, when no local identity assertion is available" (though that would be a bit redundant with the text following the note)

Reply:
Yeah, I don't want to over-rotate on redundancy.